lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -1044,7 +1044,7 @@ def list_from_ruby(arr)
     def struct_from_ruby(hash)
       ret = Google::Protobuf::Struct.new
       hash.each do |k, v|
-        ret.fields[k] ||= value_from_ruby(v)
+        ret.fields[convert_to_utf8(k.to_s)] ||= value_from_ruby(v)
```

Comment:
Are we sure the `to_s` method will do the same thing as the REST path (which probably represents these as a JSON string instead)?